tests/test_chamber_of_deputies_dataset.py

```diff
@@ -11,37 +14,67 @@ class TestChamberOfDeputiesDataset(TestCase):
     def setUp(self):
         self.path = gettempdir()
         self.subject = ChamberOfDeputiesDataset(self.path)
+        self.years = [n for n in range(2009, date.today().year + 1)]
+
 
     @skipIf(os.environ.get('RUN_INTEGRATION_TESTS') != '1',
             'Skipping integration test')
-    def test_fetch_saves_raw_files(self):
+    def test_fetch_translate_clean_integration(self):
+        # cleaning up previous tests
+        try:
+            os.remove(os.path.join(self.path, "datasets-format.html"))
```

Comment:
I believe tearDown are inadequate since we need to guarantee deletion before we run the tests (in case the last execution was cancelled/aborted) even when `tearDown` was not executed last time. I guess `setUp` should be more appropriate, maybe with a test for `os.environ.get('RUN_INTEGRATION_TESTS') != '1'`, so we don't execute them in (probable) future unit tests.